web3/_utils/module_testing/web3_module.py

```diff
@@ -712,8 +712,8 @@ async def test_batch_requests_cancel(self, async_w3: AsyncWeb3) -> None:
         assert isinstance(block_num, int)
 
     @pytest.mark.asyncio
-    async def test_batch_requests_raises_for_common_unsupported_methods(
-        self, async_w3: AsyncWeb3, async_math_contract: "AsyncContract"
+    async def test_batch_requests_raises_for_common_unsupported_methods(  # type: ignore[override]  # noqa: E501
```

Comment:
I wonder if we should change the method names here so we're not ignoring all the types?

Reply:
Comments on lines 521-522 say they need to stay the same. I remember reviewing this when Felipe set batching up, but I don't remember why they needed to be the same.